apps/modernization-ui/src/apps/search/patient/chips/PatientChips.tsx

```diff
@@ -1,6 +1,6 @@
 import { PersonFilter } from 'generated/graphql/schema';
 import { SearchCriteriaContext } from 'providers/SearchCriteriaContext';
-import Chip from './Chip';
+import Chip from '../../../../pages/advancedSearch/components/chips/Chip';
```

Comment:
nit (non-blocking): `src` is set as the `baseUrl` so you could import this as `import Chip from 'pages/advancedSearch/components/chips/Chip';`